ddtrace/profiling/_periodic.py

```diff
@@ -73,7 +94,8 @@ def start(self):
             del threading._limbo[self]
         PERIODIC_THREAD_IDS.add(self._tident)
 
-    def join(self):
+    def join(self, timeout=None):
+        # FIXME: handle the timeout argument
```

Comment:
Out of the scope of this PR?

Reply:
Not really but it was already broken before, I'm just not fixing it here… yet.